openhands/resolver/interfaces/github.py

```diff
@@ -75,7 +75,7 @@ def get_graphql_url(self) -> str:
         if self.base_domain == 'github.com':
             return 'https://api.github.com/graphql'
         else:
-            return f'https://{self.base_domain}/api/v3/graphql'
+            return f'https://{self.base_domain}/api/graphql'
```

Comment:
```suggestion
            return f'https://{self.base_domain}/graphql'
```

Would this just be `/graphql` at the end of the domain? Does it require the additional `/api` subroute?

Reply:
Ah got it! Thanks a bunch for clarifying